CHANGELOG.md

```diff
@@ -1,3 +1,286 @@
+    kubectl -n dex annotate ingress dex meta.helm.sh/release-namespace=dex
+    ```
+
+  * Gangway
+
+    ```
+    kubectl -n gangway label ingress gangway app.kubernetes.io/managed-by=Helm
+    kubectl -n gangway annotate ingress gangway meta.helm.sh/release-name=gangway
+    kubectl -n gangway annotate ingress gangway meta.helm.sh/release-namespace=gangway
+    ```
+
+  * Metrics Server
+
+    ```
+    kubectl -n kube-system label rolebinding metrics-server-auth-reader app.kubernetes.io/managed-by=Helm
+    kubectl -n kube-system annotate rolebinding metrics-server-auth-reader meta.helm.sh/release-namespace=kube-system
+    kubectl -n kube-system annotate rolebinding metrics-server-auth-reader meta.helm.sh/release-name=metrics-server
+    ```
+
+  * httpbin
+
+    ```
+    kubectl -n httpbin label ingress httpbin app.kubernetes.io/managed-by=Helm
+    kubectl -n httpbin annotate ingress httpbin meta.helm.sh/release-namespace=httpbin
+    kubectl -n httpbin annotate ingress httpbin meta.helm.sh/release-name=httpbin
+    ```
+
+##### AWS
+
+You need to remove an asset we've updated from your assets directory:
+
+```
+rm $ASSETS_DIRECTORY/lokomotive-kubernetes/aws/flatcar-linux/kubernetes/workers.tf
+```
+
+#### Upgrading
+
+##### lokocfg syntax changes
+
+Before upgrading, make sure your lokocfg configuration follows the new v0.2.0 syntax.
+Here we describe the changes.
+
+###### DNS for the Packet platform
+
+The DNS configuration syntax for the Packet platform has been simplified.
+
+Here's an example for the Route 53 provider.
+
+Old:
+
+```hcl
+dns {
+    zone = "<DNS_ZONE>"
+    provider {
+        route53 {
+            zone_id = "<ZONE_ID>"
+        }
+    }
+}
+```
+
+New:
+
+```hcl
+dns {
+    zone     = "<DNS_ZONE>"
+    provider = "route53"
+}
+```
+
+Check out the new syntax in the [Packet configuration reference](https://github.com/kinvolk/lokomotive/blob/v0.2.0/docs/configuration-reference/platforms/packet.md#attribute-reference) for details.
+
+###### External DNS component
+
+The [`owner_id` field](https://github.com/kinvolk/lokomotive/blob/v0.2.0/docs/configuration-reference/components/external-dns.md#attribute-reference) is now required.
+
+###### Prometheus Operator component
+
+There is a specific block for Grafana now.
+
+Here's an example of the changed syntax.
+
+Old:
+
+```hcl
+component "prometheus-operator" {
+    namespace              = "<NAMESPACE>"
+    grafana_admin_password = "<GRAFANA_PASSWORD>"
+    etcd_endpoints         = ["<ETCD_IP>"]
+}
+```
+
+New:
+
+```hcl
+component "prometheus-operator" {
+    namespace = "<NAMESPACE>"
+    grafana {
+        admin_password = "<GRAFANA_PASSWORD>"
+    }
+    # etcd_endpoints is not needed anymore
+}
+```
+
+Check out the new syntax in the [Prometheus Operator configuration reference](https://github.com/kinvolk/lokomotive/blob/v0.2.0/docs/configuration-reference/components/prometheus-operator.md) for details.
```

Comment:
This currently gives 404, will it be available after release?

Reply:
Yes, after this PR is merged I will tag v0.2.0 and the link should work.